session/storage_memory_test.go

```diff
@@ -91,6 +91,14 @@ func TestStorage_Remove(t *testing.T) {
 	assert.Len(t, storage.sessions, 0)
 }
 
+func TestStorage_RemoveLast(t *testing.T) {
```

Comment:
As far as I understand, this tries to remove a non-existent session ID. How is it different from `TestStorage_Remove_Does_Not_Panic`?

Reply:
TestStorage_Remove_Does_Not_Panic performs a few deletes on existing IDs whereas this one actually tries to remove a non existing one.

Reply:
Thanks